packages/core/useDark/index.ts

```diff
@@ -55,6 +57,9 @@ export function useDark(options: UseDarkOptions = {}) {
     },
   })
 
+  if (options.initialValue)
+    mode.value = options.initialValue
```

Comment:
If you are doing so, there will be no persistent at all

Reply:
Sorry I didn't understand